subgraphs/products/products.graphql

```diff
@@ -29,6 +29,7 @@ type Product implements ProductItf & SkuItf @key(fields: "id") @key(fields: "sku
   dimensions: ProductDimension
   createdBy: User
   hidden: String
+  reviewsScore: Float! @shareable
```

Comment:
as below might be simpler to just show `@override` by itself, but this definitely works with `@shareable`
https://www.apollographql.com/docs/federation/federated-types/federated-directives/#override